### PR TITLE
Merge the develop branch into CH-1100 and resolve conflicts

### PR DESCRIPTION
**Pull request by Zarabadoo**

### CODE REVIEW

File: css/acquia_lift.navbar.css [hunk 1]
```diff
@@ -96,68 +109,81 @@
...
+#navbar-administration .navbar-tray-horizontal .acquia-lift-menu-item .acquia-lift-campaign-edit,
+#navbar-administration .navbar-tray-horizontal .acquia-lift-menu-item .acquia-lift-variation-delete,
+#navbar-administration .navbar-tray-horizontal .acquia-lift-menu-item .acquia-lift-variation-rename {
+  position: relative;
+  display: block;
   float: right;
-  margin-left: 1.667em;
+  text-align: right;
 }
-
```
Comment (eshtadc) [inline css/acquia_lift.navbar.css:112]: This removed stuff should still be there.  Needs to be added to scss along with rest of changes from https://github.com/acquia/acquia_lift/commit/b664b81b03be1ce2b8286e1936901f432425b154#diff-1
https://github.com/acquia/acquia_lift/commit/b2f53a9b63f091d4309b851cc1badb9ff7a561f2#diff-1
https://github.com/acquia/acquia_lift/commit/94e7009061911edf291795adf017b61892a7211f#diff-1

Comment (Zarabadoo) [inline css/acquia_lift.navbar.css:112]: These were some of the changes that you had added into develop then? Not something that was there before the new work in develop?

Comment (eshtadc) [inline css/acquia_lift.navbar.css:112]: These were changes that are in develop and 7.x-1.x  They were released as part of last sprint's work.

Comment (Zarabadoo) [inline css/acquia_lift.navbar.css:112]: Gotcha. I will roll them in and see how they affect some of the layout rework I did and adjust accordingly if needed.

**Pull request closed without merging**